app/assets/stylesheets/application.scss

```diff
@@ -1,11 +1,29 @@
 @import "materialize/components/color";
-$primary-color: color("blue", "lighten-2") !default;
+$primary-color: #455a64 !default;
 @import "materialize";
 
+$button-back: #FFC107;
+
 .wrapper {
   min-height: 70vh;
 }
 
+body, .body-back {
+  background-color: #B6B6B6;
```

Comment:
Color literals like `#B6B6B6` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#B6B6B6` should be written as `#b6b6b6`